### ISSUE TITLE
Bug Character creation / Deleting

### ISSUE THREAD

**Issue by Kaldurenik** [WarEmu/WarBugs#10987]
First time using Github so i'm not sure if there is a copy around. Sorry if that is the case.

Steps and issues:
Deleted a character as i wished to use that character name.
Made a new character directly after deleting that character using the same name.
The new character started out at my old characters level, with starting gear and all skills unlocked.

Edit:
I also looked around for a bit and you should not be able to use the same name to even create a character until after the server restart?

**Comment by sioding** [comment]
This should be a one timer - wasn´t able to reproduce

**Comment by Timothena** [comment]
Its just happened to me too.  As Toon Eye color on create/selection screen is different with the in game eye color.  

Its just level 1. So thinking to delete and remake it.  Then say name invaild. Waaargh!

**Comment by wargrimnir** [comment]
This is intended. Deleted characters have their name reserved for some time, usually the next server restart if it's done intentionally.

**Comment by Haggal** [comment]
A couple days ago I deleted one of my characters for a reroll. I removed and sold all the gear and sent the gold to another character, and deleted the rerolling character, which seemed to work. Upon relogging into the game later that evening, at the character selection screen, the deleted character was still there, at the rank she had at deletion. As expected, all the sold gear was not there but a set of level 1 gear had been automatically created and the character could be played at the level she was at deletion. A second deletion got the character gone for good.
